app/main.py

```diff
@@ -1,25 +1,70 @@
-from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
```

Comment:
It's a good practice to validate the inputs to your class. Consider adding checks to ensure that `min_amount` is less than or equal to `max_amount`.